x-pack/plugins/stack_alerts/server/rule_types/es_query/rule_type_params.ts

```diff
@@ -48,7 +49,12 @@ const EsQueryRuleParamsSchemaProperties = {
   // how to group
   groupBy: schema.string({ validate: validateGroupBy, defaultValue: 'all' }),
   // field to group on (for groupBy: top)
-  termField: schema.maybe(schema.string({ minLength: 1 })),
+  termField: schema.maybe(
+    schema.oneOf([
+      schema.string({ minLength: 1 }),
+      schema.arrayOf(schema.string(), { minSize: 2, maxSize: MAX_SELECTABLE_GROUP_BY_TERMS }),
```

Comment:
curious why it's a minsize of 2? Could we not have an array with 1 term?

Reply:
`multi_term` doesn't accept a single element array as terms.
`terms` is supposed to be used in that case.
Therefore group_by input sends a string if only one field is selected.